CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### Fixes
 - run migrations on backup import #3006
 
+### Changes
+- refactorings #3023
+
```

Comment:
Will this change go into 1.72.0?
Probably create an "Unreleased" section? (or don't mention it as it isn't important)